Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -30,50 +30,63 @@ open class ChatChannelListItemView: _View, ThemeProvider, SwiftUIRepresentable {
+    open private(set) lazy var mainContainer: ContainerStackView = ContainerStackView()
+        .withoutAutoresizingMaskConstraints
+        .withAccessibilityIdentifier(identifier: "mainContainer")
     
     /// This container embeded by `mainContainer` containing `topContainer` and `bottomContainer`.
     open private(set) lazy var rightContainer: ContainerStackView = ContainerStackView(
         axis: .vertical,
         spacing: 4
-    ).withoutAutoresizingMaskConstraints
+    )
+    .withoutAutoresizingMaskConstraints
+    .withAccessibilityIdentifier(identifier: "rightContainer")
 
     /// By default contains `title` and `unreadCount`.
     /// This container is embed inside `mainContainer ` and is the one above `bottomContainer`
-    open private(set) lazy var topContainer: ContainerStackView = ContainerStackView().withoutAutoresizingMaskConstraints
+    open private(set) lazy var topContainer: ContainerStackView = ContainerStackView()
+        .withoutAutoresizingMaskConstraints
+        .withAccessibilityIdentifier(identifier: "topContainer")
 
     /// By default contains `subtitle` and `timestampLabel`.
     /// This container is embed inside `mainContainer ` and is the one below `topContainer`
-    open private(set) lazy var bottomContainer: ContainerStackView = ContainerStackView().withoutAutoresizingMaskConstraints
+    open private(set) lazy var bottomContainer: ContainerStackView = ContainerStackView()
+        .withoutAutoresizingMaskConstraints
+        .withAccessibilityIdentifier(identifier: "bottomContainer")
     
     /// The `UILabel` instance showing the channel name.
     open private(set) lazy var titleLabel: UILabel = UILabel()
         .withoutAutoresizingMaskConstraints
         .withAdjustingFontForContentSizeCategory
         .withBidirectionalLanguagesSupport
+        .withAccessibilityIdentifier(identifier: "titleLabel")
     
     /// The `UILabel` instance showing the last message or typing users if any.
     open private(set) lazy var subtitleLabel: UILabel = UILabel()
         .withoutAutoresizingMaskConstraints
         .withAdjustingFontForContentSizeCategory
         .withBidirectionalLanguagesSupport
+        .withAccessibilityIdentifier(identifier: "subtitleLabel")
     
     /// The `UILabel` instance showing the time of the last sent message.
     open private(set) lazy var timestampLabel: UILabel = UILabel()
         .withoutAutoresizingMaskConstraints
         .withAdjustingFontForContentSizeCategory
         .withBidirectionalLanguagesSupport
+        .withAccessibilityIdentifier(identifier: "timestampLabel")
     
     /// The view used to show channels avatar.
     open private(set) lazy var avatarView: ChatChannelAvatarView = components
         .channelAvatarView
         .init()
         .withoutAutoresizingMaskConstraints
+        .withAccessibilityIdentifier(identifier: "avatarView")
```

Comment:
What would happen if the same identifier is used more than once on the same screen? For example, for the avatar, it's easy to name it "avatarView" both as part of the channel list item, as well as in the navigation bar. Maybe we should use less generic identifiers?

Reply:
Thanks for the comment @martinmitrevski .

Unless 2 views are in the very same view hierarchy, this naming logic won't harm. 
In the other cases, when several containers were used within the same view hiereachy, we applied different accessibility identifiers.

As in, it's always possible to change the identifiers to make them unique.

Reply:
thanks for the clarification @bielikb 👍